### DISCUSSION TITLE
Can I pass a variable to the body of a HTTP POST REQUEST generated by the BOT COMPOSER?

### DISCUSSION THREAD

**Discussion by MIkey007007** [microsoft/BotFramework-Composer#5660] [Q&A]

**Comment by azamat-tyulebayev** [comment]
Yes, you can for example, store it in:
<YOUR_BOT>/settings/appsettings.json

like this:

{
....
  "MicrosoftAppId": "",
  "YOUR_VARIABLE_HERE": "VALUE",
 ....
}

and send it like this:

${settings.YOUR_VARIABLE_HERE}

**Reply by tushaar9027** [reply]
Hi azamat-tyulebayev
I want pass the answer i got from QnA MAker to microsoft translator service i am trying like this.

[
    {"Text": ${conversation.userquery}}
]

But its not working like this , i can't store it in settings... do you know how can we do this.

**Comment by guitarguySD** [comment]
You can pass data into a json object in Composer

Using Set Properties, create a variable user.myJsonObj =json('{"title": "Info Input"}')

Then you can use set property such as user.myJsonObj.newData with an existing variable =dialog.response or something.
